hedera-node/hedera-app/src/main/java/com/hedera/node/app/throttle/NetworkUtilizationManager.java

```diff
@@ -48,16 +49,18 @@ boolean trackTxn(
     void leakUnusedGasPreviouslyReserved(@NonNull final TransactionInfo txnInfo, final long value);
 
     /**
-     * Updates the throttle requirements for the given transaction and returns whether the transaction
-     * should be throttled for the current time(Instant.now).
+     * Determines if the transaction should be throttled after the transaction handler has been executed.
+     * Currently only relevant for contract ops duration throttle.
      *
-     * @param txnInfo the transaction to update the throttle requirements for
-     * @param state the current state of the node
-     * @param consensusTime the consensus time
+     * @param txnInfo       - the transaction information.
+     * @param txnStream     - the stream builder containing the results of the transaction
+     * @param consensusTime - the consensus time
      * @return whether the transaction should be throttled
      */
-    boolean shouldThrottle(
-            @NonNull final TransactionInfo txnInfo, @NonNull final State state, @NonNull final Instant consensusTime);
```

Comment:
Did we just miss getting rid of this at some point? I don't see any usages of it, but it seems odd that this just sits in the interface without anybody calling it. Of course it wouldn't be the first time an unused method wasn't deleted, but just to check.. 

As long as it's not being used, repurposing it is fine 👍

Reply:
Yes this was not being used and so I decided to repurpose.